src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
Please use java naming convention for enums. All enums values should be in capital and in snake-case form.
Also, for better readability, declare the enums in alphabetical order (in real world application, you would order it by business value). ex.
```suggestion
    BLACK, 
    BLUE,
    RED,
    WHITE
```